tokio/src/net/tcp/listener.rs

```diff
@@ -5,6 +5,7 @@ cfg_not_wasi! {
     use crate::net::{to_socket_addrs, ToSocketAddrs};
 }
 
+use crate::util::blocking_check::check_socket_for_blocking;
```

Comment:
Very minor and not a needed change to merge, but IMO this check makes more sense in the `net` module.